electrum/gui/qt/email_notification_dialogs.py

```diff
@@ -635,6 +635,17 @@ def _subscribe(self):
 
     def _unsubscribe(self):
         self.close()
+        is_unsub = custom_message_box(
```

Comment:
I know its common for bool vars to starts with `is_` but i dont like it here.

Reply:
Done